ivy/functional/frontends/paddle/fft.py

```diff
@@ -198,6 +198,30 @@ def ihfft2(x, s=None, axes=(-2, -1), norm="backward", name=None):
         return ivy.astype(ihfft2_result, ivy.complex128)
 
 
+@to_ivy_arrays_and_back
+def ihfftn(x, s=None, axes=None, norm="backward", name=None):
+    # check if the input array is two-dimensional and real
+    if len(ivy.array(x).shape) != 2 or ivy.is_complex_dtype(x):
+        raise ValueError("input must be a two-dimensional real array")
+
+    # cast the input to the same float64 type so that there are no backend issues
+    x_ = ivy.astype(x, ivy.float64)
+
+    ihfftn_result = 0
+    # Compute the complex conjugate of the 2-dimensional discrete Fourier Transform
+    if norm == "backward":
+        ihfftn_result = ivy.conj(ivy.rfftn(x_, s=s, axes=axes, norm="forward"))
+    if norm == "forward":
+        ihfftn_result = ivy.conj(ivy.rfftn(x_, s=s, axes=axes, norm="backward"))
+    if norm == "ortho":
+        ihfftn_result = ivy.conj(ivy.rfftn(x_, s=s, axes=axes, norm="ortho"))
+
+    if x.dtype in [ivy.float32, ivy.int32, ivy.int64]:
+        return ivy.astype(ihfftn_result, ivy.complex64)
+    if x.dtype == ivy.float64:
+        return ivy.astype(ihfftn_result, ivy.complex128)
+
```

Comment:
Hey @Younggregs, the implementation is a direct copy of ihfft2. Both are two different functions, you can find some idea about ihfttn [here](https://pytorch.org/docs/stable/generated/torch.fft.hfftn.html#torch.fft.hfftn)

Reply:
Hello @Dsantra92 that's what it says actually, that the only difference between the two are the default parameters.. [here](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/fft/ihfft2_en.html) also it's a Paddle function not Pytorch

Reply:
Yes, you need to remove the 2 dimensional assertions required for ihftt2.

Reply:
Alright thank you on it